### PR TITLE
update libvirt-vnc cert & key when executing enroll certificates

### PR DESCRIPTION
**Pull request by dangel101**
Bug-Url: https://bugzilla.redhat.com/2043146
Bug-Url: https://bugzilla.redhat.com/2056021

### CODE REVIEW

**Review by mwperina — CHANGES_REQUESTED** [review]
Please provide a description in PR and commit what was changed and why and please also attach relevant bug.

File: packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-vnc-certificates/meta/main.yml [hunk 1]
```diff
@@ -1,2 +0,0 @@
-dependencies:
-  - ovirt-host-deploy-facts
```
[the remark this answers is not in the document]
Comment (dangel101) [reply packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-vnc-certificates/meta/main.yml:2]: Actually I'm not sure why it was ever needed,
We don't use this dependency for the other certificates, and it works without it

File: packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-vnc-certificates/tasks/main.yml [hunk 2]
```diff
@@ -12,7 +12,6 @@
     copy:
       src: "{{ host_deploy_spice_tls_x509_cert_dir }}/{{ item }}"
       dest: "{{ host_deploy_vnc_tls_x509_cert_dir }}/{{ item }}"
-      force: no
```
[the remark this answers is not in the document]
Comment (dangel101) [reply packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-vnc-certificates/tasks/main.yml:15]: without it, it changes 'notBefore' to the current date

**Review by ahadas — APPROVED** [review]
infra know the details better, from virt point of view it makes sense to enroll the vnc certificate as well :)

File: packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-vnc-certificates/meta/main.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (dangel101) [reply packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-vnc-certificates/meta/main.yml:2]: Actually I'm not sure why it was ever needed,
We don't use this dependency for the other certificates, and it works without it

File: packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-vnc-certificates/tasks/main.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (dangel101) [reply packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-vnc-certificates/tasks/main.yml:15]: without it, it changes 'notBefore' to the current date

[the remark this answers is not in the document]
Comment (mwperina) [reply packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-vnc-certificates/tasks/main.yml:15]: Ahh, now I get when I looked at ansible documentation, `force: no` means that files are copied only when they don't exist:

https://docs.ansible.com/ansible/latest/collections/ansible/builtin/copy_module.html#parameter-force

File: packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-vnc-certificates/meta/main.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (mwperina) [reply packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-vnc-certificates/meta/main.yml:2]: OK

**Review by mwperina — APPROVED** [review]
+1

**Pull request merged**